Dockerfile

```
@@ -1,10 +1,8 @@
 FROM ubuntu:latest
 
-
 ENV CONDA_ENV_NAME=graph
 ENV PYTHON_VERSION=3.10
```

Since the docker is being updated anyway, might be worth updating the python version as well.

```suggestion
ENV PYTHON_VERSION=3.12
```